packages/manager/modules/bm-cluster-components/src/cluster-dashboard/dashboard.constants.js

```diff
@@ -0,0 +1,8 @@
+export const TRAVAUX_LINK = {
```

Comment:
do not use french variable
```suggestion
export const WORK_IN_PROGRESS_LINK = {
```